src/components/Users/UserListAndCard.tsx

```diff
@@ -49,10 +49,12 @@ export const UserStatusIndicator = ({
       ) : (
         <Badge
           variant="secondary"
-          className="bg-gray-100 whitespace-nowrap text-wrap"
+          className="bg-gray-100 inline-flex items-center gap-1 px-2 py-1 rounded-lg text-xs sm:text-sm text-gray-700 
+        max-w-full truncate overflow-hidden whitespace-nowrap"
```

Comment:
I pretty sure a few classes here are not required!

Also why do we have new line in the className?